src/Microsoft.DocAsCode.Build.Engine/TemplatePreprocessors/TemplateUtility.cs

```diff
@@ -38,5 +38,20 @@ public string ResolveSourceRelativePath(string originPath, string currentFileOut
                 return originPath;
             }
         }
+
+        public string UpdateHref(string originalHref, string sourceFileKey)
```

Comment:
get it, it's better to have better naming.

---
In reply to: [110823857](https://github.com/dotnet/docfx/pull/1532#discussion_r110823857) [](ancestors = 110823857)